lab_3_ann_retriever/main.py

```diff
@@ -49,6 +52,18 @@ def calculate_distance(query_vector: Vector, document_vector: Vector) -> float |
 
     In case of corrupt input arguments, None is returned.
     """
+    if not (isinstance(query_vector, (list, tuple)) and isinstance(document_vector, (list, tuple))
```

Comment:
Are you sure? Looks solid

Reply:
i think, there were some problems with unit tests without check whether vector is a list. 
removed